daemon/image_service.go

```diff
@@ -65,7 +65,7 @@ type ImageService interface {
 
 	// Windows specific
 
-	GetLayerFolders(img *image.Image, rwLayer layer.RWLayer) ([]string, error)
+	GetLayerFolders(img *image.Image, rwLayer layer.RWLayer, containerID string) ([]string, error)
```

Comment:
I'd be _thrilled_ if someone points out a better thing to pass into this API that can provide what's needed for both graphdriver and snapshotter implementations.